dashboard/app/views/plc/enrollment_unit_assignments/show.html.haml

```diff
@@ -26,5 +28,5 @@
     All learning resources
   - @enrollment_unit_assignment.plc_course_unit.get_all_possible_learning_resources.each do |learning_resource|
     %i{class: 'fa ' + (learning_resource.icon.nil? ? 'fa-book' : learning_resource.icon)}
-    = learning_resource.name
+    = link_to learning_resource.name, "http://#{learning_resource.resource_url}", target: '_blank'
```

Comment:
What about `https://` sites?  It might be better to let the creator of the learning resource set the full URL.